python/grass/jupyter/utils.py

```diff
@@ -60,7 +60,7 @@ def reproject_region(region, from_proj, to_proj):
     proc.stdin = None
     proj_output, stderr = proc.communicate()
     if proc.returncode:
-        raise RuntimeError("reprojecting region: m.proj error: " + stderr)
+        raise RuntimeError(f"Encountered error while running m.proj: {stderr}")
```

Comment:
This is definitively needed to avoid `TypeError: can't concat str to bytes`.

It still leaves the issue of bytes unresolved. It will show something like: `"Encountered error while running m.proj: b'ERROR: Bad things...'"`, i.e., the error message will be in `b'...'` and it will work well for English/ASCII characters only, but put here a note about a need to decode `stderr` for now.